src/main/resources/kibana-rest.yml

```diff
@@ -27,18 +27,37 @@ kibana_alerts_health:
     ">= 7.11.0 < 7.13.0": "/api/alerts/_health"
     ">= 7.13.0": "/api/alerting/_health"
 
-kibana_detection_engine_find:
+# Calculates health of Detection Engine and returns a health snapshot.
+# Scope: the whole cluster = all detection rules in all Kibana spaces.
+# https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/detection_engine/rule_monitoring/api/detection_engine_health/README.md
+kibana_detection_engine_health_cluster:
   versions:
-    ">= 7.6.0": "/api/detection_engine/rules/_find?sort_field=enabled&sort_order=asc"
+    ">= 8.8.2": "/internal/detection_engine/health/_cluster"
+
+# Calculates health of Detection Engine and returns a health snapshot.
+# Scope: all detection rules in the current/default Kibana space.
+# https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/detection_engine/rule_monitoring/api/detection_engine_health/README.md
+kibana_detection_engine_health_space:
+  versions:
+    ">= 8.8.2": "/internal/detection_engine/health/_space"
```

Comment:
Here as well I checked the payload. Lots of data there and might be good to get some enablement on that API call not that we add this API.

Reply:
> Lots of data there and might be good to get some enablement on that API call not that we add this API.

@111andre111 Do you mean some enablement for support engineers? I think it would make a lot of sense to do 👍

We're overloaded right now with urgent fixes and leftovers for 8.9.0. 8.10 is going to be a tough release cycle because of Serverless. I hope in 8.10 or 8.11 (more realistically) I'll be able to carve some time for writing documentation on how to interpret and use responses from this endpoint, and then we could set up a zoom call.